packages/react/src/components/SearchField/SearchField.test.tsx

```diff
@@ -150,6 +150,31 @@ test('should support name prop', () => {
   expect(input).toHaveDisplayValue('bananas');
 });
 
+test('SearchField should render trailingChildren as a component', async () => {
```

Comment:
These tests and the other tests below don't actually validate that the component is rendered correctly since it should be a child of `TextFieldWrapper`.

What we likely want here is to check that the expected component contains the element:

```jsx
const input = render()
expect(input.parentElement.contains(screen.getByText('1 of 1'))).toBeTruthy() 
```

I would also write the tests more like the actual intended usage, so it makes the patterns more clear.